docs/security/trust/trust_key_mng.md

```diff
@@ -11,19 +11,34 @@ parent= "smn_content_trust"
+trust makes use of five different types of keys:
 
 | Key                 | Description                                                                                                                                                                                                                                                                                                                                                                         |
 |---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | root key         | Root of content trust for a image tag. When content trust is enabled, you create the root key once. |
-| target and snapshot | These two keys are known together as the "repository" key. When content trust is enabled, you create this key when you add a new image repository. If you have the root key, you can export the repository key and allow other publishers to sign the image tags.    |
-| timestamp           | This key applies to a repository. It allows Docker repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side.                                                                                                              |
-
-With the exception of the timestamp, all the keys are generated and stored locally
-client-side. The timestamp is safely generated and stored in a signing server that
-is deployed alongside the Docker registry. All keys are generated in a backend
+| target           | This key allows you to sign image tags, to manage delegations including delegated keys or permitted delegation paths. |
+| snapshot         | This key signs the current collection of image tags, preventing mix and match attacks.
+| timestamp        | This key allows Docker repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side. |
+| delegation       | Delegation keys are optional and allow you to delegate signing image tags to other publishers without having to share your repository key |
+
+The target and snapshot key are known together as the "repository key".
+
+With the exception of the timestamp, snapshot, and delegation keys, all the keys
+are generated and stored locally client-side. Delegation keys are optional, and
+[need to be generated and added to the repository separately](trust_delegation.md#generating-delegation-keys).
+The timestamp and snapshot keys are safely generated and stored in a signing server
+that is deployed alongside the Docker registry. These keys are generated in a backend
 service that isn't directly exposed to the internet and are encrypted at rest.
 
+Prior to Docker Engine 1.11, the snapshot key was also generated and stored locally
+client-side.  Starting with Docker Engine 1.11, the snapshot key is generated and stored
```

Comment:
I think "Prior to Docker Engine 1.11, the snapshot key was also generated and stored locally client-side." should be moved to a "note",

Also, "Starting with Docker Engine 1.11" can be removed; the documentation describes the *current* version, which is 1.11 when this is published, so adding that information is redundant.